ansible/roles/k8s_cluster/defaults/main.yml

```diff
@@ -1,6 +1,23 @@
+# default values required
 k8s_version: 1.14.9
 remove: false
 remote: true
 
 ingress_host: "{{ ansible_ssh_host }}.nip.io"
 install_docker: true
+
+firewalld_public_ports:
+  - 6443/tcp
+  - 10250/tcp
+  - 10255/tcp
+  - 8472/udp
+  - 30000-32767/tcp
+
+firewalld_public_services:
+  - http
+  - https
+
+pod_subnet: 10.244.0.0/16
+service_subnet: 10.96.0.0/12
+
+coreos_flannel_sha_commit: 2140ac876ef134e0ed5af15c65e414cf26827915
```

Comment:
Why do you include such a change within this PR which is focused to add the command which offer the possibility to run the playbook locally on the host ?

Reply:
Because the installation of k8s will fail if these variables are not defined, and localhost doesn't inherit group variables